libs/utility/src/security/user.ts

```diff
@@ -46,6 +47,34 @@ export class User {
   
       return user;
     }
-    
-  }
-  
\ No newline at end of file
+
+    // TODO - Verify this mapping: JWT token from AWS Cognito converted to User.
+    static convertAwsCognitoJwtToUser(jwt: any): User {
```

Comment:
I'd expect unit tests written for this method.

Reply:
Right, a unit test is a good idea! Will do.

Reply:
Unit tests added.